docs/OnTrack/Data Analytics Platform/angular.json

```diff
@@ -0,0 +1,162 @@
+import { Component, VERSION } from '@angular/core';
```

Comment:
This is a jsx file, this is not a valid json format. Also in this repo, we only need to upload documents not the code file. Please upload your code into your own repo and make a document here to link to your code repo. Thank you.